modules/cnf-latency-tests-discovery-mode.adoc

```diff
@@ -10,7 +10,7 @@ Use discovery mode to validate the functionality of a cluster without altering i
 
 [IMPORTANT]
 ====
-When running the latency tests, **always** run the tests with `-e DISCOVERY_MODE=true` and `-ginkgo.focus` set to the appropriate latency test. If you do not run the latency tests in discovery mode, your existing live cluster performance profile configuration will be modified by the test run.
+When running the latency tests, **always** run the tests with `-ginkgo.focus` set to the appropriate latency test. 
```

Comment:
I think that now this "important" section is not relevant, because if they don't set the ginkgo focus var it still will run the latency suite and could not hurt.
Having the specific focus options in the sections below is enough imo.
But I wonder whether should we edit this message to alert the users, that if they'r using an older image (4.14 or prior) then they must have those vars. it makes sense to me, but I don't know what's the standard.

Reply:
I'll explain myself in other way:
Now that we only run the latency test suite, it's not a must to use the `-ginkgo.focus`,
so I believe we can remove this note completely.